src/gui/qgscolorwidgets.cpp

```diff
@@ -511,9 +511,26 @@ void QgsColorWheel::createImages( const QSizeF size )
 
 void QgsColorWheel::resizeEvent( QResizeEvent *event )
 {
+  QgsColorWidget::resizeEvent( event );
+#ifdef Q_OS_WIN
```

Comment:
I'm curious, as I couldn't find it in the docs, is this macro about the host or the target machine?
I mean, will it work for the mingw build which runs on fedora?

Reply:
It refers to the compilation target, so also works with MinGW on a non-Windows host.